.github/images/rabbitmq.txt

```diff
@@ -0,0 +1 @@
+docker.io/apecloud/rabbitmq:3.13.2-management
```

Comment:
Add an extra blank line at the end.
add rabbitmq to .github/workflows/save-images.yml  inputs options
add rabbitmq chart to.github/charts/kubeblocks-enterprise.txt